docs/machine-learning/automate-training-with-cli.md

```diff
@@ -28,10 +28,11 @@ You can generate those assets from your own datasets without coding by yourself,
 
 Currently, the ML Tasks supported by the ML.NET CLI are:
 
-- classification (binary and multi-class)
+- classification
 - regression
 - recommendation
-- Future: other machine learning tasks such as image-classification, ranking, anomaly-detection, clustering
+- image classification
+- Future: other machine learning tasks such as ranking, anomaly detection, clustering
```

Comment:
I wonder if we even need this future bullet point. It's not in the product now, so would remove it and just list what the product supports. Sounds more like a promise and might create confusion.

Reply:
Yes, docs shouldn't mention future plans according to the contributor's guide. @briacht Could you update and then we'll merge?